source/installation-guide/wazuh-agent/wazuh-agent-package-hpux.rst

```diff
@@ -12,38 +12,23 @@ The installed agent runs on the endpoint you want to monitor and communicates wi
 
 #. To start the installation process, download the `HP-UX installer <https://packages.wazuh.com/|WAZUH_CURRENT_MAJOR_HPUX|/hp-ux/wazuh-agent-|WAZUH_CURRENT_HPUX|-|WAZUH_REVISION_HPUX|-hpux-11v3-ia64.tar>`_. 
 
+.. note:: You need root user privileges to run all the commands described below.
+
```

Comment:
Fix list numbers (there is two number 1), also, the `Note` should be before the "To start the installation..."

![image](https://user-images.githubusercontent.com/14913942/220918629-6262d595-efcc-466f-b806-ddb26208d8e5.png)

Reply:
The idea was to reduce the scope of `root` requirement

Reply:
In other deployments it is specified before starting the process, we must be consistent with the rest of the documentation, if you consider that it should change, open an issue so the @wazuh/content can work on it.

![image](https://user-images.githubusercontent.com/14913942/220990519-e1d5dc6a-e81c-49c1-9911-5b10700d714a.png)